### ISSUE TITLE
Make instr/data bus info available on RVFI

### ISSUE THREAD

**Issue by Silabs-ArjanB** [openhwgroup/cv32e40x#547]
Feed signals like prot, memtype back through prefetcher with rvalid/instr timing, so that they can be exported with 'instruction timing' on RVFI. Will help in (formal) verification of the bus signals. Such additional prefetcher content should get optimized away in the RTL.

**Comment by silabs-robin** [comment]
Maybe also other signals like `dbg`.

E.g.:
```
assert property (
  rvfi_valid && rvfi_custom.dbg  // Fetched as D-mode
  |->
  rvfi_custom.prot[2:1] == 2'b 11  // Fetched as M-mode
);
```

**Comment by silabs-oysteink** [comment]
This should now be resolved by PR #857
